src/main/java/baseball/Application.java

```diff
@@ -1,7 +1,122 @@
 package baseball;
 
+import java.util.Arrays;
+import java.util.Random;
+import java.util.Scanner;
+import java.util.stream.Stream;
+
 public class Application {
     public static void main(String[] args) {
-        // TODO: 프로그램 구현
+
+        Scanner sc = new Scanner(System.in);
+
+        // 컴퓨터 수 생성
+        int comNum = getComputerNumber();
+        int[] comDigits = Stream.of(String.valueOf(comNum).split(""))
+                .mapToInt(Integer::parseInt)
+                .toArray();
+
```

Comment:
computerNumber 또는 computer로 fullname을 적어주는 게 컨벤션에 맞을 것 같아요!

Reply:
123을 [1,2,3]으로 바꾸는 연산 정도는 속도(?)를 위해 stream 대신 기본 연산을 활용하는 건 어떨까요?

```suggestion
comDigits.add(number % 10) ;
number %= number / 10 ;
...
comDigits.reverse();
```